tests/credential_plugins_test.py

```diff
@@ -162,3 +165,89 @@ def test_tss_import(self) -> None:
         ):
             # assert this module as opposed to older thycotic.secrets.server
             assert cls.__module__ == 'delinea.secrets.server'
+
+
+@pytest.mark.parametrize(
+    (
+        'url',
+        'app_id',
+        'object_query',
+        'object_query_format',
+        'reason',
+        'verify',
+    ),
+    [
+        (
+            'http://testurl.com',
+            'foobar123',
+            'foobar123',
+            'test',
+            'foobar123',
+            True,
+        ),
+        (
+            'http://testurl.com',
+            'foobar123',
+            'foobar123',
+            'test',
+            '',
+            True,
+        ),
+    ],
+)
+def test_aim_sensitive_traceback_masked(
+    mocker: MockerFixture,
+    monkeypatch: pytest.MonkeyPatch,
+    url: str,
+    app_id: str,
+    object_query: str,
+    object_query_format: str,
+    reason: str | None,
```

Comment:
Looking into the params, I see that it's always `str`. There's no test that checks `None`.

```suggestion
    reason: str,
```

Or did you want to add another param with `None` additionally to an empty string?

Reply:
I think just an empty str actually covers it. change reverted